pyproject.toml

```diff
@@ -41,10 +41,7 @@ dev = [
 ]
 
 [project.scripts]
-demo-deploy = "offspot_demo.deploy:entrypoint"
```

Comment:
I though we'd keep the individual scripts for dev/testing. That's why in deploy and prepare we have entrypoints that sets debug for instance

Reply:
I'm fine with it, I understood the opposite yesterday. Let me add them back them, and add one for toggle.

Reply:
I'm not sure I was clear myself about what I wanted/expressed 😅 I know I said we'd have only 2 entrypoints (setup, watcher) – actually 3– but in my mind I meant this as public entrypoints that we'll use on prod.
With current change, I cannot test parts independently without writing some code… that I will have to keep separately

Reply:
Yup, makes sense